src/components/LearnMore.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import logopic from '../img/blank.png';
+
+export default function LearnMore() {
+  return (
+    <Container style={{ textAlign: 'left', paddingTop: '50px' }}>
+      <Grid container spacing={2}>
+        <Grid item>
+          <img width="500" src={logopic} className="profilePic" alt="" />
+        </Grid>
+        <Grid item xs={12} sm={6} container>
+          <Grid item xs container direction="column" spacing={2}>
```

Comment:
For vertical centering try `style={{ alignItems:'center' }}`
I believe it should work for the first container but if not maybe try it on the the one below.

Reply:
Right that does work for horizontal but I was messing around with the web inspector and discovered that align-self: 'center' does the centering for vertical position. Thanks for the help btw